contracts/dapis/DapiServer.sol

```diff
@@ -667,6 +689,10 @@ contract DapiServer is
     /// Airnodes without requiring a request or subscription. The Beacons for
     /// which the fulfillment data and signature is omitted will be read from
     /// storage.
+    /// @dev This signed data here is specific to this contract, which is to be
+    /// used when the signer does not want to provide the more general
+    /// signature. EIP712 may feel relevant here, but we avoided it for the
```

Comment:
Probably dumb question but which is "the more general signature"?

Reply:
I mean the one from `updateDataFeedWithSignedData()`